### PR TITLE
solution

### PR DESCRIPTION
**Pull request by ghost**
- [DEMO LINK](https://ostUp.github.io/layout_calendar/)

### CODE REVIEW

**Review by HusakOleh — APPROVED** [review]

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length_31">
+      <div class="calendar__day calendar__day--start-day_sun"></div>
```
Comment (dellachaise) [inline src/index.html]: You should implement start-day modifier for the calendar block.

Change modifiers names to a complex modifier.

Make sure, that you check your task according to the checklist before pull request.

Screenshot 2022-07-22 at 10 36 28

```suggestion
    <div class="calendar calendar--month-length--31 calendar__day--start-day--sun">
      <div class="calendar__day "></div>
```

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  margin: 0 auto;
```
Comment (dellachaise) [inline src/styles/blocks/calendar.scss]: Don't add margin to a block, only for element. Use mix.
 More details in video.
[video, timing 4:15](https://mate.academy/learn/css-advanced/bem#/video/911)

File: src/styles/blocks/calendar.scss [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i}) {
+
+        &::before {
+          content: "#{$i}";
+        }
+
+        &:hover {
+          cursor: pointer;
```
Comment (dellachaise) [inline src/styles/blocks/calendar.scss]: Move cursor: pointer to &__day

Comment (dellachaise) [inline src/styles/blocks/calendar.scss]: move :hover styles from cycle for to line 23(24)

File: src/styles/blocks/calendar.scss [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: $number-size;
+    border: 1px solid black;
+    background-color: $cell-color;
+    transition: transform 0.5s, background-color 0.5s;
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i}) {
+
+        &::before {
```
Comment (dellachaise) [inline src/styles/blocks/calendar.scss]: Better write this selector in 1 line, kind of this: `&:nth-child(#{$i})::before`

**Review by dellachaise — CHANGES_REQUESTED** [review]
Good start 👍

Please, add fixes according to comments.

**Review by dellachaise — APPROVED** [review]
Well done 👍

**Pull request closed without merging**